spyderlib/plugins/shortcuts.py

```diff
@@ -318,43 +292,65 @@ def check_conflicts(self):
                 conflicts.append(shortcut)
         return conflicts
 
-    def update_warning(self, reset=False):
-        """ """
-        conflicts = self.check_conflicts()
-        if reset:
-            conflicts = []
-
+    def update_warning(self, warning_type=NO_WARNING, conflicts=[]):
+        """Update warning label to reflect conflict status of new shortcut"""
         widget = self.helper_button
-
-        if conflicts:
-            tip_title = _('The new entered shorcut conflicts with:') + '\n'
+        widget_message = self.label_warning
+
+        if warning_type == NO_WARNING:
+            warn = False
+            tip = '<br><br>'
+        elif warning_type == SEQUENCE_CONFLICT:
+            template = '<i>{0}<b>{1}</b></i>'
+            tip_title = _('The new shorcut conflicts with:') + '<br>'
             tip_body = ''
             for s in conflicts:
-                tip_body += ' - {0}: {1}\n'.format(s.context, s.name)
-            tip = '{0}{1}'.format(tip_title, tip_body)
+                tip_body += ' - {0}: {1}<br>'.format(s.context, s.name)
+            tip = template.format(tip_title, tip_body)
+            warn = True
+        elif warning_type == SEQUENCE_LENGTH:
+            template = '<i>{0}</i>'
+            tip = _('A compound sequence can have {break} a maximum of '
+                    '4 subsequences.{break}').format(**{'break': '<br>'})

```

Comment:
Type `Ctrl+1, Ctrl+2, Ctrl+3, Ctrl+4, Ctrl+5` (all while holding control and in quick succession) This shortcut has more than for and is not accepted, hence it will display and error.